build/stackmat.js

```diff
@@ -317,7 +317,11 @@
       this.start = bind(this.start, this);
       this.signalFetched = bind(this.signalFetched, this);
       if (!supported()) {
-        alert("You need a recent browser in order to connect your Stackmat Timer.");
+        if (options != null ? options.onNonSuppotedBrowser : void 0) {
```

Comment:
Yeah, having generated code in the repository is annoying. I usually would be in favor of removing `build`, but wouldn't this be annoying for anyone wanting to use the old "c&p library into local directory and be done with it" method?

(I didn't know about the `prepublish` hook. Nice.)